src/pages/CyborgGuide.tsx

```diff
@@ -1,9 +1,267 @@
-function CyborgGuide() {
+import { Col, Container } from 'react-bootstrap'
+import styled from 'styled-components'
+import { Navbar } from '../components/Navbar'
+import Canary from '../static/mock-canary.svg'
+
+const CyborgGuide = ({
+  setActiveAccount,
+  activeAccount,
+}: {
+  setActiveAccount: (activeAccount: string) => void
+  activeAccount: string
+}): JSX.Element => {
   return (
-    <div>
-      <h2>CyborgGuide</h2>
-    </div>
+    <>
+      <Container>
+        <Navbar
+          showAccount
+          showBrandIcon
+          setActiveAccount={setActiveAccount}
+          activeAccount={activeAccount}
+        />
+      </Container>
+      <Container>
+        <GuideRow>
+          <GuideTitle xs lg="6" className="display-1 text-uppercase pt-5">
+            Cyborg Guide
+          </GuideTitle>
+          <Col className="text-uppercase text-end ms-auto mt-4">
+            <p>
+              <a
+                target="_blank"
+                href="https://wiki.polkadot.network/docs/maintain-guides-society-kusama"
+                rel="noreferrer"
+              >
+                Kusama society guide
+              </a>
+            </p>
+            <p>
+              <a
+                target="_blank"
+                // eslint-disable-next-line max-len
+                href="https://polkascan.io/kusama/transaction/0x948d3a4378914341dc7af9220a4c73acb2b3f72a70f14ee8089799da16d94c17"
+                rel="noreferrer"
+              >
+                Tattoo rules
+              </a>
+            </p>
+          </Col>
+        </GuideRow>
+      </Container>
+      <Container>
+        <GuideRow>
+          <Col>
+            <div className="h1 text-uppercase font-weight-bold">
+              The journey
+            </div>
+            <p className="mb-1">
+              To become a Cyborg means to get a membership in the Kappa Sigma Mu
+              Society.
+            </p>
```

Comment:
I think we should start thinking about i18n

Reply:
Added a card to Trello.